### ISSUE TITLE
Next and Previous buttons fail color contrast ratio

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#121]
URL: https://hcdev.openplus.ca/en/alert-recall/test-certain-happy-veggie-world-hearts-choices-and-vegefarm-brand-vegan-and-1
User Agent: Firefox 93.0
Language: Both

Issue Description: Non text content “arrow” image inside the previous, next icons fail contrast ratio at default and focused state against their adjacent colour, which is the icon background colour.
Default = #FF9261 (2.2:1)
Focused = #FF6D2C  (2.8:1)

![image](https://user-images.githubusercontent.com/68117281/137160872-94a412ac-31cb-4095-8a38-fb41363bcfb0.png)

Steps to reproduce: Select the url above and use the Color Contrast Analyzer

Expected results: Elements in a graphic that communicate meaning must have a contrast ratio of at least 3:1.

**Comment by echouin** [comment]
Sent to OP

**Comment by echouin** [comment]
@ndolter OP indicates that this is fixed :)

**Comment by ndolter** [comment]
Validated that the color contrast now passes.
